src/components/Icon/Expensicons.js

```diff
@@ -10,6 +10,7 @@ import ChatBubble from '../../../assets/images/chatbubble.svg';
 import Checkmark from '../../../assets/images/checkmark.svg';
 import Clipboard from '../../../assets/images/clipboard.svg';
 import Close from '../../../assets/images/close.svg';
+import Cash from '../../../assets/images/cash.svg';
```

Comment:
Looks like we keep this in dictionary order. Same for `Paypal`.
Not related to this PR if you can also place `NewWorkspace` and `ExpensifyCard` in correct place so we maintain an order.

Reply:
Yup

Reply:
I think you forgot to change `Cash` and `Paypal`  😄

Reply:
Check now.